python/ray/util/sgd/torch/training_operator.py

```diff
@@ -318,6 +367,25 @@ def optimizers(self):
         """List of optimizers created by the ``optimizer_creator``."""
         return self._optimizers
 
+    @property
+    def train_loader(self):
+        """
+        Data loader for the validation dataset created by the ``data_creator``.
+        """
```

Comment:
BTW, if there is only one sentence, it should be on one line - 
```suggestion
        """Data loader for the validation dataset created by the ``data_creator``."""
```

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
It does not fit in 79 characters